setuptools/command/bdist_wheel.py

```diff
@@ -220,7 +220,7 @@ class bdist_wheel(Command):
 
     def initialize_options(self) -> None:
         self.bdist_dir: str | None = None
-        self.data_dir: str | None = None
+        self.data_dir = ""
```

Comment:
`data_dir` is *always* set in `finalize_options`, so if I understand correctly this value doesn't matter and it'll always be a string.